packages/core/admin/webpack.config.js

```diff
@@ -91,6 +91,7 @@ module.exports = ({
           options: {
             loader: 'tsx',
             target: buildTarget,
+            jsx: 'automatic',
```

Comment:
I was surprised this wasn't inferred from the tsconfig, but that's webpack for you – this means we don't have to `import React from 'react'` if we're not using any of it's exports 🙌🏼